app/controllers/application_controller.rb

```diff
@@ -22,15 +21,9 @@ class ApplicationController < ActionController::API
   ].freeze
 
   before_action :block_unknown_hosts
```

Comment:
Are we ok with the block_unknown_hosts  action happening after authentication?

Reply:
oh... hmmm, i didn't think about that. Good catch!